docs/pages/database-access/auto-user-provisioning/postgres.mdx

```diff
@@ -90,6 +94,78 @@ Users created within the database will:
 
 (!docs/pages/includes/database-access/auto-user-provisioning/postgres15-grant-create.mdx!)
 
+### User does not have CONNECT privilege error
+
+You may encounter the following error when the admin user or the roles assigned
+to the auto-provisioned user do not have the permission to connect to the target
+database:
+
+```shell
+> tsh db connect mysql --db-name <database> example
+...
+FATAL: permission denied for database "<database>"
+DETAIL: User does not have CONNECT privilege.
+```
+
+Make sure `CONNECT` is granted to the admin user and the respective roles:
+```sql
+GRANT CONNECT ON DATABASE <database> to "teleport-admin";
+GRANT CONNECT ON DATABASE <database> to "reader";
+```
+
+### Cannot execute in a read-only transaction error
+
+You may encounter the following error when connecting to an AWS RDS Aurora
+reader endpoint:
+```shell
+> tsh db connect mysql --db-name <database> example
+psql: error: connection to server at "localhost" (::1), port 12345 failed: Connection refused
+...
+ERROR: cannot execute CREATE ROLE in a read-only transaction (SQLSTATE 25006)
+```
+
+Database auto-user provisioning is curerntly not compatible with RDS Aurora
```

Comment:
```suggestion
Database auto-user provisioning is currently not compatible with RDS Aurora
```

Reply:
also, do we have any plans to add support for RDS aurora reader endpoints? If not, maybe just drop the "currently" altogether.

Reply:
Could we provision the user against primary endpoint?

Reply:
> also, do we have any plans to add support for RDS aurora reader endpoints? If not, maybe just drop the "currently" altogether.

good point. will remove "currently".

> Could we provision the user against primary endpoint?

We probably can look into this in the future to provision the user against the primary endpoint first then connect the user to the reader. Though we will have to see how much time it will take to propagate the users/roles change from primary to replicas.

For tracking: https://github.com/gravitational/teleport/issues/37530